app/src/test/java/com/android/shelflife/utils/DateUtilsUnitTest.kt

```diff
@@ -114,6 +114,15 @@ class DateUtilsUnitTest {
     assertEquals("01012020", dateStr)
   }
 
+  @Test
+  fun testFormatTimestampToDisplayDate() {
+    val sdf = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
+    val date = sdf.parse("01/01/2020")
+    val timestamp = Timestamp(date!!)
+    val dateStr = formatTimestampToDisplayDate(timestamp)
+    assertEquals("01/01/2020", dateStr)
```

Comment:
**(nit-picky):** the test is good, but you could have used a different month-date combination to make sure it is the proper dd/MM format and not MM/dd, since currently your test does not insure the European date format
```suggestion
  fun testFormatTimestampToDisplayDate() {
    val sdf = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
    val date = sdf.parse("18/06/2020")
    val timestamp = Timestamp(date!!)
    val dateStr = formatTimestampToDisplayDate(timestamp)
    assertEquals("18/06/2020", dateStr)
```